packages/react-native/cli.js

```diff
@@ -107,17 +114,30 @@ The behavior will be changed on ${chalk.white.bold(CLI_DEPRECATION_DATE.toLocale
 }
 
 function warnWithDeprecated() {
-  if (process.argv[2] !== 'init') {
+  if (!isInitCommand) {
     return;
   }
 
   console.warn(`
-${chalk.yellow('⚠')}️ The \`init\` command is deprecated.
+${chalk.yellow('🚨')}️ The \`init\` command is deprecated.
```

Comment:
Nit: I don't think you can color 🚨:
```suggestion
🚨 The \`init\` command is deprecated.
```

Reply:
Sorry I missed this. You're right it's a total noop.